.travis.yml

```diff
@@ -103,12 +103,15 @@ install:
   - export PATH="$HOME/miniconda/bin:$PATH"
   - conda create -q --yes -n test python=$PYTHON_VERSION
   - source activate test
-  - conda install -q --yes numpy=$NUMPY_VERSION sphinx || conda install -q --yes -c conda-forge numpy=$NUMPY_VERSION sphinx
+  - conda install -q --yes numpy=$NUMPY_VERSION || conda install -q --yes -c conda-forge numpy=$NUMPY_VERSION
   - conda install -q --yes -c conda-forge gsl
+  - python -m pip install --upgrade pip
+  - python -m pip install --ignore-installed certifi --upgrade
+  - python -m pip install sphinx>=1.8
   - if [[ "${INSTALL_LIBGFORTRAN}" == "TRUE" ]]; then conda install -q --yes libgfortran ; fi
   - make -r CC=$COMPILER
   - make install CC=$COMPILER
-  - python setup.py install
+  - python -m pip install -e .
```

Comment:
`pip install -e` is a great trick, I can't believe I'm just learning about it now!

Reply:
Yup - it's very handy. It's the recommended way for testing while developing/tinkering